.changelogs/function-name-correction.yml

```diff
@@ -0,0 +1,7 @@
+significance: patch
+type: fixed
+links:
+  - "#2550"
+entry: Updated the wrong function name in
+  `includes\functions\llms.functions.templates.dashboard.php` to
+  `lifterlms_student_dashboard`.
```

Comment:
```suggestion
entry: Fixed checking for the wrong function name when 
  defining the the pluggable function `lifterlms_student_dashboard`.
```

Reply:
Got it. Thanks for the correction. 
Note: Pushing a new commit due to 2x `the`